bumblebee_status/modules/contrib/system.py

```diff
@@ -77,7 +77,7 @@ def popup(self, widget):
             util.cli.execute(popupcmd)
             return
 
-        menu = util.popup.menu()
+        menu = util.popup.menu(self.__config)
```

Comment:
This is the best I came up with for passing on the config object to the popup menu. WDYT?